### PR TITLE
Unable to create domain objects

### PR DESCRIPTION
**Pull request by scottbell**
Fixes: https://github.com/nasa/openmct/issues/4666 and https://github.com/nasa/openmct/issues/4674

@scottbell @shefalijoshi after this is merged into master, we should cherrypick this back into 1.8.3?

### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [ ] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [ ] Command line build passes?
* [ ] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### CODE REVIEW

**Comment by scottbell** [comment]
This fix addresses https://github.com/nasa/openmct/issues/4666

**Comment by scottbell** [comment]
> @scottbell i'm going to be out of pocket for a little while. Would you mind pulling these changes into your PR? #4671
> 
> This should prevent this class of regression in the future.

Done

**Review by unlikelyzero — APPROVED** [review]
Looks great! I'll wait for a openmct developer to sign off on the changes

File: package.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
     "test:debug": "cross-env NODE_ENV=debug karma start --no-single-run",
     "test:coverage": "cross-env NODE_OPTIONS=\"--max_old_space_size=4096\" COVERAGE=true karma start --single-run",
     "test:coverage:firefox": "cross-env NODE_OPTIONS=\"--max_old_space_size=4096\" karma start --single-run --browsers=FirefoxHeadless",
-    "test:e2e:ci": "npx playwright test --config=e2e/playwright-ci.config.js smoke",
+    "test:e2e:ci": "npx playwright test --config=e2e/playwright-ci.config.js smoke default condition.e2e",
```
Comment (unlikelyzero) [inline package.json:99]: @nikhilmandlik this adds new suites to the PR checks

**Comment by jvigliotta** [comment]
@scottbell can we remove #4666 from this one, I believe this one is related to #4623 (not directly, as that one is more styles based, but it's in the same ballpark) I think @shefalijoshi has a fix for the overall domainObject issue (we can't  create ANY objects).

**Review by jvigliotta — APPROVED** [review]
Nice work!

### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue?? | Y
Appropriate unit tests included? | Y (existing)
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (if PR closes issue) | Y

**Pull request merged**